charts/rancher-monitoring/101.1.0+up40.1.2/charts/hardenedKubelet/templates/pushprox-servicemonitor.yaml

```diff
@@ -8,6 +8,13 @@ metadata:
   labels: {{ include "pushProxy.serviceMonitor.labels" . | nindent 4 }}
 spec:
   endpoints: {{include "pushProxy.serviceMonitor.endpoints" . | nindent 4 }}
+    {{ if .Values.global.cattle.clusterId }}
+    - metricRelabelings:
+      - action: replace
+        sourceLabels: [__address__]
+        targetLabel: cluster_id
+        replacement: {{ .Values.global.cattle.clusterId }}
+    {{- end }}
```

Comment:
I don't think this works. We'd need to add it to **each** endpoint, not define a separate endpoint that does metricRelabelings.

The fix here should be put into the template within `_helpers.tpl`.